spec/acceptance/suites/README.md

```diff
@@ -0,0 +1,121 @@
+# simp-core Acceptance and integration tests
+
+In this directory there are tests that can be used to test SIMP in it's
+entirety, including all non ISO components.
+
```

Comment:
which of these tests run 'simp config' and 'simp bootstrap'?  Some do, and some manually affect bootstrapping.